cmake/Modules/opm-upscaling-prereqs.cmake

```diff
@@ -27,5 +27,6 @@ set (opm-upscaling_DEPS
 	dune-grid REQUIRED;
 	opm-common REQUIRED;
 	opm-core REQUIRED;
+        opm-output REQUIRED;
 	dune-cornerpoint REQUIRED"
```

Comment:
> (dune-cornerpoint does not require opm-output)

That's what I said.  Did you mean output does not require cornerpoint?  If so, that's certainly true in a technical sense (no code in output uses functionality from cornerpoint), but semantically speaking output requires grids and aren't we supposed to merge core and cornerpoint Soon[tm]?